### ISSUE TITLE
[Feature Request][Low priority] New icon for Hidden apps in settings menu

### ISSUE THREAD

**Issue by jcperil** [falzonv/discreet-launcher#265]
This is a very low priority request. Right now, the icon for **Hidden apps** in the settings menu is a scissor, which I think is not very intuitive.

![scissor](https://user-images.githubusercontent.com/89805719/210086578-97a3a017-6038-4f1c-ae4c-d637a33f2728.jpg)
If you agree, I would recommend something like below, adjusted with the colors of the menu, of course.
![hide](https://user-images.githubusercontent.com/89805719/210083969-487ce91e-85ae-4324-ba5c-e493030858a8.jpg)
Credit: NewPipe

**Comment by falzonv** [comment]
Hello,

Thank you for the suggestion, I agree this icon is not very intuitive.
My initial idea was to stay with Android built-in icons and none of them was like the crossed eye above so I took the scissors.

Later I noticed that built-in icons looked quite different in Android 5 so I decided to embed these icons in the app so that they looked the same on all versions but I didn't thought about improving this icon.

I will have a look for a replacement like you suggested but in the same design that the existing icons (colors, border).

Best regards.
